### PR TITLE
Phone keypad problem

### LINKED ISSUE #486: Content: Phone keypad problem
## 📚 Documentation

Given an integer array containing digits from [0, 9], the task is to print all possible letter combinations that the numbers could represent.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Both codes in C++ and python are written by me

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #486

### CODE REVIEW

**Comment by gayatri517** [comment]
Please review my PR @iamrajiv @atarax665 @gat786 @nidh-ish
Thank you

**Comment by gayatri517** [comment]
Please review my PR @iamrajiv @atarax665 @gat786 @nidh-ish
Thank you

**Comment by gayatri517** [comment]
Please review my PR @iamrajiv @ankitaggarwal23 @atarax665 @gat786 @nidh-ish @siddharth25pandey @bhav09
Thank you

**Comment by gayatri517** [comment]
@atarax665 Please review PR

File: docs/competitive-programming/phone-keypad.mdx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- Initialize an empty array(out)
+    - If the input string reaches its end
+        - Add a NULL character at the end
+	- Print the string
+
+    - digit=input[i]-'0'
+    - make a for loop from 0 to keypad[digit][k] is not NULL
+	- include the current character in the out array
+	- Make a recursive call for furthur combinations
+
```
Comment (siddharth25pandey) [inline docs/competitive-programming/phone-keypad.mdx:57]: Write this concept of Algorithm before Giving examples

File: docs/competitive-programming/phone-keypad.mdx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+<Tabs
+  defaultValue="cpp"
+  values={[
+    { label: 'C++', value: 'cpp', },
+    { label: 'Python', value: 'py', },
+  ]
+}>
+<TabItem value="cpp">
+
+```cpp
```
Comment (siddharth25pandey) [inline docs/competitive-programming/phone-keypad.mdx:72]: Write C++ not cpp

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@gayatri517 Do the suggested changes

**Comment by gayatri517** [comment]
@siddharth25pandey I have made the changes and resolved the merge conflict. Kindly review it

**Review by siddharth25pandey — APPROVED** [review]
@gayatri517 LGTM 💯 Able to merge

**Pull request merged**